scripts/validate_analysis_record.py

```diff
@@ -0,0 +1,44 @@
+import validate_record
+import utils
+
+ANALYSES_FIELDS_CONVERSION_MAPPING = {
+    'datasetAccession': 'study',
+    'sampleAccessions': 'samples'
+}
+FIELDS_TO_BE_REMOVED = [
+    'accession',  # validation tool use fixed id field
+    # not in the ruleset
+    'standardMet',
+    'versionLastStandardMet',
+    'releaseDate',
+    'updateDate',
+    'fileSizes',
+    'organism',
+    'urls',
+    'datasetInPortal'
+]
+logger = utils.create_logging_instance("validate_analysis")
+
+
+class validate_analysis_record(validate_record.validate_record):
```

Comment:
class name should use CamelCase convention

Reply:
fixed in 2db90ac